specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
Yes, I just wanted to attempt to capture what a standard summary report looks like. If/when we can, we'll refine it to accurately capture that which different vendors' equipment emits.